apps/src/templates/projects/PersonalProjectsTable.jsx

```diff
@@ -35,101 +37,14 @@ export const COLUMNS = {
   ACTIONS: 5
 };
 
-export const styles = {
```

Comment:
Moved styles and other helper methods to the bottom so that the main component was easier to distinguish